pallets/nonfungible/src/common.rs

```diff
@@ -125,6 +110,16 @@ impl<T: Config> CommonWeightInfo<T::CrossAccountId> for CommonWeights<T> {
 	}
 }
 
+pub(crate) fn mint_with_props_weight<T: Config>(
+	create_no_data_weight: Weight,
+	tokens: impl Iterator<Item = u32> + Clone,
```

Comment:
```suggestion
	token_properties: impl Iterator<Item = u32> + Clone,
```
Не сразу сообразил что это, пришлось в вызов смотреть
По хорошему ко всем этим вычислительным методам нужны doccomments, поскольку из-за того как оно размазано очень сложно следить за логикой